<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editBusDialog</class>
 <widget class="QDialog" name="editBusDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit bus</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>321</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_19">
    <item>
     <widget class="QLabel" name="label_55">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="model_B"/>
    </item>
    <item>
     <widget class="QLabel" name="label_57">
      <property name="text">
       <string>Company Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="comname_B"/>
    </item>
    <item>
     <widget class="QLabel" name="label_58">
      <property name="text">
       <string>Organization Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="orgnr_B"/>
    </item>
    <item>
     <widget class="QLabel" name="label_59">
      <property name="text">
       <string>Bus type:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="bustyp"/>
    </item>
    <item>
     <widget class="QLabel" name="label_60">
      <property name="text">
       <string>Inside Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="insnr">
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_62">
      <property name="text">
       <string>Number of seats:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="nrofsea">
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>editBusDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>editBusDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
